Cargo.lock

```diff
@@ -1,3209 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
I don't think you should delete this file. It version locks our dependencies. If you update any dependencies or (I assume) change the dependency graph in the workspace, there will be changes to this file which you should simply commit.

Reply:
Done, thanks!